apps/src/applab/themeValues.js

```diff
@@ -64,7 +94,22 @@ const themeValues = {
       bubblegum: '#FFC3E6',
       millennial: '#FEDBD0',
       robot: '#D8E0E3',
-      classic: '#FFFFFF'
+      classic: '#FFFFFF',
+      coralReef: '#5BA0ED',
+      mintChip: '#5CC8D7',
+      lavender: '#A189BA',
+      cherryVanilla: '#FFF5F5',
+      berryPatch: '#64337D',
+      cucumber: '#AAEBDF',
+      crushedVelvet: '#46154A',
+      playtime: '#FCFF5E',
```

Comment:
Nit: I'm randomly selecting theme/attribute combos to check the color on and this one is listed as 'FCFF5D' in Figma. I'm not sure it matters because it's so close, but I figured I'd make a note regardless.

Reply:
talked offline about this but I used the actual values on the design, not the description, as the source of truth when there were discrepancies.